### PR TITLE
[#108537412] Add filtering option

### PR DESCRIPTION
**Pull request by tadejm**
More details in https://www.pivotaltracker.com/n/projects/1320848

:ab: add filtering params by property type

### CODE REVIEW

**Comment by patientfrenzy** [comment]
Looks good to me :+1:

File: lib/ht/search_client/remote.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@ class Remote
       :order, :per_page, :types, :features,
       :bedrooms, :page, :guests, :price_min,
       :price_max, :currency, :aggregations, :range_aggregations,
-      :excluding_ids
+      :excluding_ids, :ib, :ib_ee
```
Comment (mrship) [inline lib/ht/search_client/remote.rb]: Do these needs to be abbreviations?

Comment (tadejm) [inline lib/ht/search_client/remote.rb]: They don't need to be. We decided to stick use abbreviations for the AB test. If we decide to productionize we'll use more human friendly language.

Comment (mrship) [inline lib/ht/search_client/remote.rb]: :+1:

**Comment by mrship** [comment]
:shipit:

**Pull request merged**